stories/HorizontalProgressBar/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import './OverrideStyle.scss';
+import HorizontalProgressBar from '../../src/components/HorizontalProgressBar/index.js';
+
+const WrappedProgressBar = props => <HorizontalProgressBar {...props} />;
+
+storiesOf('HorizontalProgressBar', module).add('default', () => (
+  <WrappedProgressBar />
+));
+
+storiesOf('HorizontalProgressBar', module).add('half way progress', () => (
+  <WrappedProgressBar totalProgress={100} currentProgress={50} />
+));
+
+storiesOf('HorizontalProgressBar', module).add('override style', () => (
+  <div className="override-progress-bar">
+    <WrappedProgressBar totalProgress={100} currentProgress={50} />
+  </div>
+));
```

Comment:
can we get another story where we change the  progress with a click of a button? (to see the animation).

also, can we not eliminate the totalProgress prop and make the entire component percentage based? so we only ever pass 1 'progress' percentage prop?